### ISSUE TITLE
was nsff tested with 360 captured scene?

### ISSUE THREAD

**Issue by nischal-sanil** [zhengqili/Neural-Scene-Flow-Fields#21]
Were any experiments conducted with 360 captured scene? Curious to know if there is any difference between training forward facing vs 360 scene.

**Comment by owang** [comment]
Yes we didn’t experiment with these, I’d like to try it out though. The issue with 360 dynamic scene capture is that you can presumably not make correspondences across 180 degrees of rotation (e.g., that will be too long in time, so your deforming object has probably deformed a lot), so its unlikely that you could pause the scene and then rotate entirely around it. although maybe with the followup work on on neural trajectory fields this becomes possible. Also, for objects that stay near a rest pose, a canonical approach might be more suitable for 360 rotation.
